cmd/upgrade.go

```diff
@@ -2,10 +2,16 @@ package cmd
 
 import (
 	"fmt"
+	"io"
+	"os"
 
+	"github.com/exercism/cli/cli"
+	"github.com/exercism/cli/debug"
 	"github.com/spf13/cobra"
 )
 
+var out io.Writer
```

Comment:
I might've borrowed it 😉 

Funny enough, I was reading up on testing in the GPL book and came across it for testing command lines, so good call.